packages/fsshopify/src/normalizers/blogPosts.ts

```diff
@@ -0,0 +1,25 @@
+export interface Article {
+  author: string;
+  image: string;
+  description: string;
+  title: string;
+  publishedAt: string;
+  url: string;
+  id: string;
+  isArticle: boolean;
+  excerpt: string;
+}
+
+export function blogPosts(blogData: any): Article {
+  return {
+    author: blogData.node.authorV2.name,
+    image: blogData.node.image.originalSrc,
+    description: blogData.node.contentHtml,
+    excerpt: blogData.node.excerpt,
+    title: blogData.node.title,
+    publishedAt: blogData.node.publishedAt,
+    url: blogData.node.url,
+    id: blogData.node.id,
```

Comment:
Should also add checks here.

Reply:
I am not sure how many people will want to consume the blog feed so maybe we should remove this functionality all together and really just focus in ecom/user related features.

Reply:
@Cauldrath @jasonmosley PR updated.